backends/arm/test/visualize.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 Arm Limited and/or its affiliates.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import logging
+from typing import Optional
+
+from executorch.backends.arm.test.common import arm_test_options, get_option
+from torch.export import ExportedProgram
+
+logger = logging.getLogger(__name__)
+_model_explorer_installed = False
+
+try:
+    # pyre-ignore[21]: We keep track of whether import succeeded manually.
```

Comment:
Do you just test locally?

Reply:
Yeah, this has only been tested locally. I am not sure how to test in ci since it requires an external dependency

Reply:
@Olivia-liu - what do you think of this vs. our SDK plans for visualization?

Reply:
Bump @Olivia-liu, some kind of visualization would be great, so would really appreciate merging unless you have other plans soon

Reply:
@Erik-Lundell Sorry about the delay! This is great. Glad to see integration with model-explorer going on!